### PR TITLE
Getting jetpack through remote

### PR DESCRIPTION
**Pull request by BannZay**
There comes some refactoring code. Hopefully its not problem?

### CODE REVIEW

File: src/game/server/ddracecommands.cpp [hunk 1]
```diff
@@ -166,6 +166,13 @@ void CGameContext::ConRifle(IConsole::IResult *pResult, void *pUserData)
 	pSelf->ModifyWeapons(pResult, pUserData, WEAPON_RIFLE, false);
 }
 
+void CGameContext::ConJetpack(IConsole::IResult *pResult, void *pUserData)
+{
+	CGameContext *pSelf = (CGameContext *)pUserData;
+	CCharacter* pChr = pSelf->GetPlayerChar(pResult->m_ClientID);
+	pChr->m_Jetpack = true;
```
Comment (Henningstone) [inline src/game/server/ddracecommands.cpp]: Possibly unsafe, `pChr` may be a nullptr

File: src/game/server/ddracecommands.cpp [hunk 2]
```diff
@@ -190,6 +197,13 @@ void CGameContext::ConUnRifle(IConsole::IResult *pResult, void *pUserData)
 	pSelf->ModifyWeapons(pResult, pUserData, WEAPON_RIFLE, true);
 }
 
+void CGameContext::ConUnJetpack(IConsole::IResult *pResult, void *pUserData)
+{
+	CGameContext *pSelf = (CGameContext *)pUserData;
+	CCharacter* pChr = pSelf->GetPlayerChar(pResult->m_ClientID);
+	pChr->m_Jetpack = false;
```
Comment (Henningstone) [inline src/game/server/ddracecommands.cpp]: same as above

**Comment by Henningstone** [comment]
You may also consider changing the commit message, I think it doesn't quite fit ;)
`git commit --amend`

**Comment by def-** [comment]
Thanks!

**Pull request merged**